src/utils.ts

```diff
@@ -0,0 +1,76 @@
+import { Transform, TransformOptions, Writable } from "stream";
+
+export type HashMap<T> = { [k: string]: T }
+
+export class ConsoleWriter extends Writable {
+    constructor() {
+        super({objectMode: true});
+    }
+
+    _write(chunk: any, encoding: string, callback: Function): void {
+        console.log(chunk);
```

Comment:
Nigdy nie używaj bezpośrednio console.log(), bo wpływa na performence, dlatego, że jest to funkcja synchroniczna, więc spowalnia inne wydarzenia asynchroniczne